index.html

```diff
@@ -52,6 +56,100 @@ <h2 class="title2">I'm a Software Developer</h2>
+            <button class="btn">See Project</button>
+        </div>
+    </section>
+
+    <section class="project">
+        <div class="my-cards container">
+            <div class="card">
+                <div class="card-content">
+                    <h2>Profesional Art Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <button>See project</button>
+            </div>
+
+            <div class="card">
+                <div class="card-content">
+                    <h2>Profesional Art Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <button>See project</button>
+            </div>
+
+            <div class="card">
+                <div class="card-content">
+                    <h2>Profesional Art Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <button>See project</button>
+            </div>
+
+            <div class="card">
+                <div class="card-content">
+                    <h2>Profesional Art Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <button>See project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about-section">
+        <div class="about-me container">
+            <h2>About Me</h2>
+            <p>
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and
+                experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <button class="btn">Get my resume</button>
+        </div>
+    </section>
+
```

Comment:
- [ ] You didn't implement the categories in the about me section. Please implement this as it is part of the requirement for this milestone. 👋 
> Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.

- Screenshot:
![Screenshot 2023-03-31 230026](https://user-images.githubusercontent.com/107632780/229241758-a3e2f5af-5976-41b9-8933-e9c179358516.png)

Reply:
categories implement in the "about me" section